### ISSUE TITLE
Code review: Footer in contact page

### ISSUE THREAD

**Issue by LL835** [fac26/Georgia-and-Karol#11]
Hey, website looks amazing! We noticed that the footer in the contact page is not attached to the bottom of the page. We think it might be because of the height in the root element? It seemed to work ok without it. (We weren't sure why this happened even though the footer was set to 'position: absolute' and 'bottom: 0')
```
:root {
	font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
	text-rendering: optimizeLegibility;
	-webkit-font-smoothing: antialiased;
	-moz-osx-font-smoothing: grayscale;
  height: 1500px;
}
```

**Comment by ko-karol** [comment]
Hey. Thanks a lot! I included the height attribute temporarily because there is not much on the page and I wanted it to be scrollable so you could see Georgia's navbar moving.  

`position:absolute` is different from `position:fixed` which is what I assume you were thinking of. Absolute keeps the footer in the same position on the page but fixed will (also keep it in the same position) but move with the page scrolling.

As for `bottom: 0;`, that's just a leftover from testing 😂

**Comment by Kr33L** [comment]
Closes #16
